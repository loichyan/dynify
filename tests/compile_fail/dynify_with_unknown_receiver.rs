#[dynify::dynify]
trait Trait {
    async fn test(self: MySelf);
}

fn main() {}
