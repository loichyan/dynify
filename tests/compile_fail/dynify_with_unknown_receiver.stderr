error: unsupported receiver type
 --> tests/compile_fail/dynify_with_unknown_receiver.rs:3:19
  |
3 |     async fn test(self: MySelf);
  |                   ^^^^
